realm/functions/get_token/source.js

```diff
@@ -0,0 +1,48 @@
+exports = async function(){
+  const GOOGLE_TOKEN_ENDPOINT = "https://oauth2.googleapis.com/token";
+
+  const CLIENT_ID = context.values.get("GOOGLE_CLIENT_ID");
+  const CLIENT_SECRET = context.values.get("GOOGLE_CLIENT_SECRET");
+  
+  const tokens_collection = context.services.get("mongodb-atlas").db("auth").collection("auth_tokens");
+  
+  // Look up tokens:
+  let tokens = await tokens_collection.findOne({_id: "youtube"});
```

Comment:
Is this a YouTube-specific function?  If so, should we name it or add a comment to indicate it's for YouTube and not generic?

Reply:
My plan is to make it generic. So the name's probably correct, but it'll take a parameter that determines which service it's authenticating against.